config/routes.rb

```diff
@@ -228,6 +228,8 @@
     resources :searchgov_domains, concerns: :active_scaffold do
       member do
         post 'reindex'
+        get 'confirm_delete'
+        post 'delete_domain'
```

Comment:
Shouldn't we able to delete the domain doing a `DELETE` request to the searchgov_domains url from the resources setting above?